README.md

```diff
@@ -1,41 +1,73 @@
+# OpenSWE1R
+*An Open-Source port of the 1999 Game ["Star Wars: Episode 1 Racer"](https://en.wikipedia.org/wiki/Star_Wars_Episode_I:_Racer)*
+
+OpenSWE1R is a number of projects related to "Star Wars: Episode 1 Racer".
+We foster a community which focuses on understanding and enhancing the original game.
+
+Our main project is an Open-Source port of the game to the C programming language.
 
 ---
 
-### OpenSWE1R
-*An Open-Source port of the 1999 Game ["Star Wars: Episode 1 Racer"](https://en.wikipedia.org/wiki/Star_Wars_Episode_I:_Racer)*
+<p align="center">
+<a href="https://gitter.im/OpenSWE1R"><img alt="Gitter Chat" src="https://img.shields.io/badge/Gitter-Chat-red.svg"></a>
+<a href="https://cla-assistant.io/OpenSWE1R/openswe1r"><img alt="CLA assistant" src="https://cla-assistant.io/readme/badge/OpenSWE1R/openswe1r"></a>
+<a href="https://ci.appveyor.com/project/JayFoxRox82949/openswe1r/branch/master"><img alt="AppVeyor CI" src="https://img.shields.io/appveyor/ci/JayFoxRox82949/openswe1r/master.svg?label=AppVeyor%20CI"></a>
+<a href="https://travis-ci.org/OpenSWE1R/openswe1r"><img alt="Travis CI" src="https://img.shields.io/travis/OpenSWE1R/openswe1r/master.svg?label=Travis%20CI"></a>
+</p>
+
+---
 
-![Screenshot of OpenSWE1R running on Linux](https://i.imgur.com/LAqHPgh.png)
+<p align="center">
+<img alt="Screenshots of OpenSWE1R running on Linux" src="http://jannikvogel.de/scratchpad/openswe1r.svg" />
+</p>
 
-### Details
+---
+
+### Community
+
+OpenSWE1R hosts [a chat on gitter.im](https://gitter.im/OpenSWE1R). There is also an [IRC bridge](https://irc.gitter.im/).
 
-This project is a mix of console emulation and something like [WINE](https://www.winehq.org/).
-It implements the subset of Windows functions used by the game.
-OpenSWE1R also contains x86 CPU emulation.
-The original games code is replaced by open-source code which is patched into the game at runtime.
-This is similar to the approach used by [OpenRCT2](https://openrct2.website/), but more emphasis is put on emulation and platform independence.
+You only need a GitHub or Twitter account to access the chat.
 
-Currently, not all functions are implemented in the open-source portion.
-Therefore, this project also still relies on the original games code.
-Even if not all code is patched, the game will still work cross-platform due to the API and CPU abstraction.
+The chat is used discuss all OpenSWE1R projects, but we also talk about other aspects of "Star Wars: Episode 1 Racer".
+
+We also have [a Wiki](https://github.com/OpenSWE1R/openswe1r/wiki).
+
+---
 
-You will still need the original game for its art assets: levels, sounds, ...
+To make the Open-Source port possible, we have to do a lot of research.
+Therefore OpenSWE1R is split into smaller projects and repositories.
```

Comment:
Reword this paragraph, or change the order.
This might be the first line people read, so it should not start out with "To make <what we previously talked about>"